tools/analysis/analysis_runner.py

```diff
@@ -71,9 +114,14 @@ def main():
             results_bucket=args.results)
 
   elif args.name:
-    _request(
-        args.name, args.ecosystem, args.version, local_file=args.file,
-        results_bucket=args.results)
+    if args.all:
```

Comment:
nit: We might want to support --all for other ways ot specifying packages too. e.g. via "--list --all".

Reply:
Good idea. Added support